client/src/main/scala/ch/epfl/bluebrain/nexus/iam/client/IamClient.scala

```diff
@@ -105,6 +110,68 @@ class IamClient[F[_]] private[client] (
       else F.pure(false)
     }
 
+  /**
+    * It applies the provided function ''f'' to the ACLs Server-sent events (SSE)
+    *
+    * @param f      the function that gets executed when a new [[AclEvent]] appears
+    * @param offset the optional offset from where to start streaming the events
+    */
+  def aclEvents(f: AclEvent => F[Unit], offset: Option[String] = None)(implicit E: Effect[F],
```

Comment:
I think it's ok to make `F: Effect[F]` in the class constructor and drop the `E: Effect[F]` implicits for the functions.